test/utils/getCliCommand.ts

```diff
@@ -0,0 +1,3 @@
+export function getCliCommand() {
```

Comment:
I was trying out some ways to run the compiled one directly. It is a remnant of that. Lemme remove it.

Reply:
Just FYI with `npm run cli` it takes about twice the time to run. We need to figure out the issue of having es6 and node way of import in the same repo. Typescript allows for compiling into es6 imports but it is experimental right now.

Reply:
right ~ when we have this built and is a more user-facing CLI (e.g. `rill start`) it will be executing against a built version, which should speed things up.